lib/simbricks/base/if.c

```diff
@@ -152,6 +152,13 @@ size_t SimbricksBaseIfSHMSize(struct SimbricksBaseIfParams *params) {
 
 int SimbricksBaseIfInit(struct SimbricksBaseIf *base_if,
                         struct SimbricksBaseIfParams *params) {
+  /* a sync interval larger than the latency will lead to incorrect behaviour */
+  if (params->link_latency < params->sync_interval) {
+    fprintf(stderr,
+            "SimbricksBaseIfListen: latency must be larger or equal to sync"
```

Comment:
Wrong function name: should be `SimbricksBaseIfInit` instead of `SimbricksBaseIfInit`